packages/hagrid/hagrid/cli.py

```diff
@@ -2532,9 +2551,31 @@ def quickstart_cli(
             )
 
         if url:
-            file_path, _ = quickstart_download_notebook(
-                url=url, directory=directory, reset=reset
-            )
+            allowed_schemes_as_url = ["http", "https"]
+            url_scheme = urlparse(url).scheme
+
+            if url_scheme not in allowed_schemes_as_url:
+                notebooks = get_urls_from_dir(
+                    repo=repo, branch=branch, commit=commit, url=url
+                )
+                if not reset:
+                    overwrite_all_notebooks = click.confirm(
```

Comment:
We might have to define a default value to `overwrite_all_notebooks=True` as this variable will be undefined at line:2572 when `reset=True`.

Reply:
Thanks for that help, totally overlooked it! Fixed via https://github.com/OpenMined/PySyft/pull/6686/commits/4d9a30b35d7ea9197cf50f10f41c7f470d2134d0